src/main/js/EditorPage.jsx

```diff
@@ -112,11 +112,22 @@ SaveDialog.propTypes = {
 
 @observer
 class PipelineLoader extends React.Component {
-    state = {}
-    
+    constructor(props) {
+        super(props);
+        this.state = { scmId: 'github' };
+    }
+
     componentWillMount() {
         pipelineStore.setPipeline(null); // reset any previous loaded pipeline
-        this.loadPipeline();
+        const { organization, pipeline } = this.props.params;
+        let href = Paths.rest.pipeline(organization, pipeline);
```

Comment:
fetchPipeline expects an href, so perhaps that needs to be changed to allow doing that. I do not plan to do it in this PR.